test/e2e-eventing-upgrade-tests.sh

```diff
@@ -33,25 +33,11 @@
 
 export GO111MODULE=auto
 
-source $(dirname $0)/e2e-common.sh
-
-readonly EVENTING_READY_FILE="/tmp/prober-ready-eventing"
-readonly EVENTING_PROBER_FILE="/tmp/prober-signal-eventing"
-
-# TODO: remove when components can coexist in same namespace
-export TEST_EVENTING_NAMESPACE=knative-eventing
-export E2E_UPGRADE_TESTS_SERVING_USE=false
-# FIXME(ksuszyns): remove when knative/operator#297 is fixed
-export E2E_UPGRADE_TESTS_SERVING_SCALETOZERO=false
```

Comment:
Please, restore those lines to be sure no problem are present. They should be fixed in knative/operator#297.